templates/partners/partial/_footer.html

```diff
@@ -1,30 +1,11 @@
-<section class="p-section l-footer--sticky">
-  <div class="row">
+<section class="p-strip--white is-deep l-footer--sticky">
```

Comment:
Do we need `l-footer--sticky`? Nothing happens if I remove it.

Reply:
Nope, removed!